log/logger.go

```diff
@@ -0,0 +1,54 @@
+// Package log provides beautiful logger for the console interface.
+package log
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	clr "github.com/TwiN/go-color"
```

Comment:
Does it automatically disable colors on older (non-ANSI) terminals?

Reply:
No, but I'm wondering whether any practical use-case (developer's terminal or CI) is non-ANSI

Reply:
Yeah, probably not. I'm fine if you ignore it.